src/hooks/useMultiplayerState.ts

```diff
@@ -154,20 +151,20 @@ export function useMultiplayerState({
         toast.info("Asset uploading is disabled");
         return false;
       }
+
       if (file.size > envs!.TLDRAW__ASSETS_MAX_SIZE) {
-        toast.info(
-          `Asset is too big - max. ${
-            envs!.TLDRAW__ASSETS_MAX_SIZE / 1048576
-          }MB`,
-        );
+        const bytesInMb = 1048576;
+        const sizeInMb = envs!.TLDRAW__ASSETS_MAX_SIZE / bytesInMb;
+        toast.info(`Asset is too big - max. ${sizeInMb}MB`);
         return false;
       }
 
       const fileExtension = file.name.split(".").pop()!;
-      if (
+      const isExtensionDisallowed =
         envs!.TLDRAW__ASSETS_ALLOWED_EXTENSIONS_LIST &&
```

Comment:
you do not check by file extension but by mime type

Reply:
yeah, makes more sense, I was 'inspired' by how tldraw does this under the hood but I changed it now

also, do you mind checking trivy-vulnerability-scanning job, it has been failing for a few days now I think